dev/faq/tidb.md

```diff
@@ -39,7 +39,7 @@ TiDB 目前还不支持触发器、存储过程、自定义函数、外键，除
 
 #### 1.1.7 TiDB 具备高可用的特性吗？
 
-TiDB 天然具备高可用特性，TiDB、TiKV、PD 这三个组件都能容忍部分实例失效，不影响整个集群的可用性。具体见 [TiDB 高可用性](/key-features/#高可用)。
+TiDB 天然具备高可用特性，TiDB、TiKV、PD 这三个组件都能容忍部分实例失效，不影响整个集群的可用性。具体见 [TiDB 高可用性](/key-features.md#高可用)。
```

Comment:
I think the markdown file will be converted to a link without suffix `.md`.

Reply:
not for now, it's converted to `https://pingcap.com/key-features/#%E9%AB%98%E5%8F%AF%E7%94%A8` which is not right

Reply:
> I think the markdown file will be converted to a link without suffix `.md`.

Yes. The link on the website does not contain `.md`, while in Markdown file, we need the `.md`.